clients/unshaded/src/main/java/tachyon/client/OutStream.java

```diff
@@ -18,50 +18,16 @@
 import java.io.IOException;
 import java.io.OutputStream;
 
-import tachyon.conf.TachyonConf;
-
 /**
- * <code>OutStream</code> is the base class of output stream for TachyonFile streaming output. To
- * get an instance of this class, one should call the method <code>getOutStream</code> of
- * <code>tachyon.client.TachyonFile</code>, rather than constructing a new instance directly in the
- * client code.
+ * Provides a stream API to write to Tachyon. Only one OutStream should be opened for a Tachyon
+ * object. This class is not thread safe and should only be used by one thread.
  */
 public abstract class OutStream extends OutputStream {
```

Comment:
I will create an interface called Cancelable, and handle this similar to Seekable